app/src/test/java/org/oppia/android/app/player/state/StateFragmentLocalTest.kt

```diff
@@ -995,6 +998,102 @@ class StateFragmentLocalTest {
     }
   }
 
+  @Test
+  @Config(qualifiers = "port")
+  fun testStateFragment_mobilePortrait_finishExploration_endOfSessionConfettiIsDisplayed() {
+    launchForExploration(FRACTIONS_EXPLORATION_ID_1).use {
+      startPlayingExploration()
+      playThroughAllStates()
+      clickContinueButton()
+
+      onView(withId(R.id.full_screen_confetti_view)).check(matches(hasActiveConfetti()))
+    }
+  }
+
+  @Test
+  @Config(qualifiers = "land")
```

Comment:
Per http://robolectric.org/device-configuration/#cumulative-qualifiers should this be "+land"? Ditto for port above.

FYI @rt4914 & @anandwana001 that we might be doing this wrong elsewhere.

Reply:
Will need to check this because tests were passing so how was that possible.

Reply:
Done, I think it does miss the `+` elsewhere. Omitting `+` just defaults the other configurations (e.g. screen size) but keeps the land / port config

Reply:
I think in this case the other tests are less sensitive to setting the display density (which is the only thing we'd lose when not using '+'). We should fix this broadly, though, since it is more correct and will make the movement to #2047 easier.